modules/admin/server/routes/admin.server.routes.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+var adminPolicy = require('../policies/admin.server.policy'),
+	admin = require('../controllers/admin.server.controller');
+
+module.exports = function(app) {
+	// Users collection routes
+	app.route('/api/users').all(adminPolicy.isAllowed)
```

Comment:
I don't think this makes more sense.  What you are suggesting is that every time someone creates a new module, they should use unique api routes for that module.  This can grow out of control and be even more confusing, especially if you are exposing your end-points to third parties.